dags/analyzer_helper/discord/fetch_discord_platforms.py

```diff
@@ -0,0 +1,65 @@
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class FetchDiscordPlatforms:
+    """
+    A class to fetch Discord platform data from a MongoDB collection.
+
+    Attributes:
+        client (MongoClient): The MongoDB client.
+        db (Database): The MongoDB database.
+        collection (Collection): The MongoDB collection.
+    """
+
+    def __init__(self, db_name="Core", collection="platforms"):
+        """
+        Initializes the FetchDiscordPlatforms class.
+
+        Args:
+            db_name (str): The name of the database. Defaults to 'Core'.
+            collection (str): The name of the collection. Defaults to 'platforms'.
+        """
+        self.client = MongoSingleton.get_instance().client
+        self.db = self.client[db_name]
+        self.collection = self.db[collection]
+
+    def fetch_all(self):
+        """
+        Fetches all Discord platforms from the MongoDB collection.
+
+        Returns:
+            list: A list of dictionaries, each containing platform data with the following fields:
+                - platform_id: The platform ID (_id from MongoDB).
+                - metadata: A dictionary containing action, window, period, selectedChannels, and id.
+                - recompute: A boolean set to False.
+        """
+        query = {"platform": "discord"}
```

Comment:
Please update the query to this one

```
{
   "disconnectedAt": None,
   "platform": "discord",
}
```

we don't want to process discord platforms that is disconnected.